src/main/java/hudson/plugins/jira/JiraRestService.java

```diff
@@ -48,7 +48,7 @@
 
     private static final Logger LOGGER = Logger.getLogger(JiraRestService.class.getName());
 
-    public static final int TIMEOUT_IN_10_SECONDS = 10000;
+    public static final int MILIS_IN_ONE_SECOND = 1000;
```

Comment:
I would not expect it ever changes (proof?), so it's OK to keep directly in the method